.github/workflows/backend-lint.yml

```
@@ -88,5 +88,14 @@ jobs:
           #      `getconf ARG_MAX` - max # bytes of args + environ for exec()
           pre-commit run --files ${{ steps.files.outputs.all_files }}
 
+      # Apply fixed files via pull request review comments
+      - name: Apply any eslint fixed files as Pull Request review comments
```

```suggestion
      - name: Apply any pre-commit fixed files as Pull Request review comments
```

Was this just bad copypaste? Also could we see an example python bot comment? You only showed it for some js files.